static/bidder-info/lockerdome.yaml

```diff
@@ -7,3 +7,7 @@ capabilities:
   site:
     mediaTypes:
       - banner
+userSync:
+  redirect:
+    url: "https://lockerdome.com/usync/prebidserver?pid="+cfg.Adapters["lockerdome"].PlatformID+"&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}"
+    userMacro: "{{uid}}"
```

Comment:
The LockerDome adapter is using the PlatformID in an unintended way. I suggest we comment out their userSync and add a comment:

```
# lockerdome requires a platform id for their user sync process. replace <<PlatformID>>
# in the url below and uncomment the userSync section to enable.
#
#userSync:
#  redirect:
#    url: "https://lockerdome.com/usync/prebidserver?pid=<<PlatformID>>&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect={{.RedirectURL}}"
#    userMacro: "{{uid}}"
```

Reply:
Modified content to comment.